.github/workflows/windows_build.yml

```diff
@@ -45,13 +46,13 @@ jobs:
               run: conda install -y winpty
             - name: Install build/test dependencies
               shell: bash -l {0}
-              run: pip install maturin toml pytest pytest-lazy-fixture flaky
+              run: pip install maturin toml "pytest<7" pytest-lazy-fixture flaky
             - name: Build pywinpty
               shell: bash -l {0}
               run: maturin develop
             - name: Run tests
               shell: pwsh
-              run: python runtests.py
```

Comment:
Can we revert to the script here?

Reply:
Done, and reverted other pytest-related changes